.github/workflows/ci.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
       - uses: cachix/install-nix-action@v13
       - name: Check Nix parsing
         run: |
-          find . -name "*.nix" -exec nix-instantiate --parse --quiet {} >/dev/null +
+          nix-shell --run evalnix
```

Comment:
`nix-shell` - so much for migration to flakes :D Can we use nix 3.0 syntax, please?